_rules/autocomplete-valid-73f2c2.md

```diff
@@ -278,4 +278,4 @@ Autocomplete attribute contains only [whitespace](#whitespace).
 
```

Comment:
Description of this example need to be updated to autocomplete contains no tokens.